src/foam/u2/wizard/data/UcjDataLoader.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
```

Comment:
We have part of this already in UserCapabilityJunctionLoader.js

Reply:
Updated UserCapabilityJunctionLoader.js instead https://github.com/kgrgreer/foam3/pull/3122/commits/e212023724429eda9ec6ebda87f7c9ff9316b3ad